saucal/pinterest-for-woocommerce
# Feature/250 api v4

### Changes proposed in this Pull Request:

Closes #250

#

- Upgrade ads/v3 to ads/v4 endpoints.
- Use new catalog APIs for merchant registration, domain verification, and catalog ingestion.


### Screenshots:


### Detailed test instructions:
1. Install the plugin.
2. Make the connection.
3. Everything should work as usual.

### Changelog entry

>
